Other/MergeShops/BoneBreakerMergeShop[MEM].cs

```diff
@@ -0,0 +1,95 @@
+using System.Runtime.ExceptionServices;
+using System.Runtime.InteropServices;
+using System.Diagnostics;
+using System.Reflection.PortableExecutable;
+/*
+name: Bonebreaker Merge
+description: This bot will farm the items belonging to the selected mode for the Bonebreaker Merge [1051] in /bonebreak
+tags: bonebreaker, merge, bonebreak, outcast, boneshattering, brokenbone, skull, bonepiercer, membership, dailies, daily
+*/
+//cs_include Scripts/CoreDailies.cs
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreAdvanced.cs
+using Skua.Core.Interfaces;
+using Skua.Core.Models.Items;
+using Skua.Core.Options;
+
+public class BonebreakerMergeMerge
+{
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+    private CoreFarms Farm = new();
+    private CoreAdvanced Adv = new();

```

Comment:
the include for dailies was added but not the private line here which is required